cfgov/retirement_api/jinja2/retirement_api/claiming.html

```diff
@@ -38,6 +38,21 @@ <h1>{{ _("Planning your Social Security claiming age") }}</h1>
                     <a href="es/">Ver página en español</a>
                     {% endif %}
                 </p>
+                <div class="o-well">
+                    {% if current_language == 'en' %}
+                    <p>The Social Security Administration offers
+                         <a class="a-link a-link__icon" href="https://www.ssa.gov/multilanguage/interpreter.html">free interpreter services</a>
```

Comment:
```suggestion
                         <a href="https://www.ssa.gov/multilanguage/interpreter.html">free interpreter services</a>
```

I thought we shouldn't need or want to explicitly include `a-link` or `a-link__icon` in source HTML, since our automatic link decoration should add that? Same comment applies to all other links here.

Reply:
Just added new commits to remove all such classes throughout the file